system/View/Cell.php

```diff
@@ -178,7 +178,8 @@ protected function determineClass(string $library): array
         }
 
         // locate and return an instance of the cell
-        $object = Factories::cells($class);
+        // @TODO extend Factories to be able to load classes with the same short name.
+        $object = class_exists($class) ? new $class() : Factories::cells($class);
```

Comment:
In v4.2.12, just `new $class()` was called:
https://github.com/codeigniter4/CodeIgniter4/blob/558db238d043ae3fd65c277df3e5e34d53773abd/system/View/Cell.php#L82-L83